src/LanguageServer.Common/Position.cs

```diff
@@ -69,7 +69,7 @@ public Position(int lineNumber, int columnNumber)
         /// <summary>
         ///     Is the position 0-based (instead of 1-based)?
         /// </summary>
-        public bool IsZeroBased { get; private set; }
```

Comment:
This is not IDE refactoring. I checked that setter isn't used anywhere, so we can delete it to make whole `Position` structure `readonly`